qiskit_machine_learning/neural_networks/effective_dimension.py

```diff
@@ -336,6 +340,7 @@ def weight_samples(self, weight_samples: Union[np.ndarray, int]) -> None:
             # there is a weird mypy error if we keep the same variable name, so there's 'weights'
             weights = np.asarray(weight_samples)
             # additional check to accept 1D arrays
+            # additional check to accept 1D arrays
```

Comment:
Did you accidentally double-up this comment?

Reply:
yes, that is a typo. Fixing it now.